### DISCUSSION TITLE
Welcome to RPi-Jukebox-RFID Discussions!

### DISCUSSION THREAD

**Discussion by MiczFlor** [MiczFlor/RPi-Jukebox-RFID#1186] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. THIS IS NOT ABOUT BUG REPORTS: please us the "Issues" thread for that. We still have to figure out how "discussions" can be used most efficiently. We just started it, let's see what it does. Discussions about the "use of Discussions" are welcome :)

We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by s-martin** [comment]
Great idea.

I've built a Phoniebox last year for Christmas for my kids. Unfortunately currently the box has a hardware defect I'm trying to fix for this year's Christmas.

Since last year I've been hanging around here trying to help other users, triaging issues, improving the code and training my limited Python skills.

I like this project, because it has a low entry barrier (if don't have too much hardware attached ;) and every Phoniebox is an original (well except mine, because @Franzformator made the design for a couple of boxes and I just used it ;).
